NewsApp/Extensions/UIImageView+Extensions.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UIImageView+Extensions.swift
+//  NewsApp
+//
+//  Created by Reuben Simphiwe Kuse on 2024/08/28.
+//
+
+import Foundation
+import UIKit
+
+extension UIImageView {
+    func loadImage(from urlString: String) {
+        guard let url = URL(string: urlString) else { return }
+        
+        URLSession.shared.dataTask(with: url) { data, response, error in
+            if let error = error {
+                print("Error loading image: \(error.localizedDescription)")
+                return
+            }
+            
+            guard let data = data else { return }
+            
+            DispatchQueue.main.async {
+                self.image = UIImage(data: data)
+            }
+        }.resume()
+    }
+}
```

Comment:
What would this extension do since you have `KingFisher` to help with fetching an image from a url and caching it?

Reply:
I know but I was just reminding myself of the  process of fetching and displaying images. But I can comment it out or delete it.